### PR TITLE
 fix(sdk_updates): Do not block on empty cache when rendering issue details

### PR DESCRIPTION
**Pull request by tonyo**
This adds a periodic Celery task that fills/updates the cache, and then we just read from it.

### CODE REVIEW

File: src/sentry/sdk_updates.py [hunk 1]
```diff
@@ -427,6 +413,8 @@ def get_suggested_updates(setup_state, index_state=None, parent_suggestions=None
 
     suggestions = list(_get_suggested_updates_step(setup_state, index_state))
 
+    logger.warn(suggestions)
```
Comment (untitaker) [inline src/sentry/sdk_updates.py:416]: can we remove this or lower the level

File: src/sentry/tasks/release_registry.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+from sentry.net.http import Session
+from sentry.tasks.base import instrumented_task
+from sentry.utils import metrics
+
+
+logger = logging.getLogger(__name__)
+
+SDK_INDEX_CACHE_KEY = u"sentry:release-registry-sdk-versions"
+APP_INDEX_CACHE_KEY = u"sentry:release-registry-app-versions"
```
Comment (untitaker) [inline src/sentry/tasks/release_registry.py:16]: Do we even use the app index anywhere?

**Review by untitaker — APPROVED** [review]

File: src/sentry/tasks/release_registry.py (hunk 2 above)
Comment (untitaker) [reply src/sentry/tasks/release_registry.py:16]: nvm saw the getsentry PR

File: src/sentry/sdk_updates.py (hunk 1 above)
Comment (tonyo) [reply src/sentry/sdk_updates.py]: Oopsie, removed, thanks

**Pull request merged**